controller-select/identify.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import re
+antimicrox_path = r"C:\Program Files\AntiMicroX\bin\antimicrox.exe"
```

Comment:
We'll need to double check this is the correct path on the machine.

Reply:
I pulled from Staging today (12/06) and it works well